gnd/src/main/java/com/google/android/gnd/ui/home/HomeScreenFragment.java

```diff
@@ -341,6 +360,19 @@ private void onApplyWindowInsets(WindowInsetsCompat insets) {
     updateBottomSheetPeekHeight(insets);
   }
 
+  private void onAddPolygon(Point point) {
+    if (polygonPoints.size() > 3) {
+      if (polygonPoints.contains(point)) {
```

Comment:
This would require the user to select the exact position containing the first vertex, which I assume would be very difficult. Can we add some tolerance to this?

@coreyleamon How close do you think the center of the map be to the first vertex to close the polygon? Would it make more sense to calculate this distance in pixels, or degrees lat/lng?